src/devices/Qmc5883l/samples/Program.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Device.I2c;
+using System.Numerics;
+using System.Threading;
+using Iot.Device.Qmc5883l;
+
+I2cConnectionSettings settings = new(1, Qmc5883l.DefaultI2cAddress);
+using I2cDevice device = I2cDevice.Create(settings);
+using (Qmc5883l sensor = new(device))
+{
+    // Configure the sensor to match our needs.
+    // Make sure to set the device in Continuous mode if you plan on reading any data.
+    sensor.DeviceMode = Mode.Continuous;
+    sensor.FieldRange = FieldRange.Gauss8;
+    sensor.Interrupt = Interrupt.Disable;
+    sensor.OutputRate = OutputRate.Rate200Hz;
+    sensor.RollPointer = RollPointer.Disable;
```

Comment:
ideally simple sample should not require anything to be set, only show those for advanced users